supplements/tashas-cauldron-of-everything/artificer-infusions.xml

```diff
@@ -246,7 +248,436 @@
+		</description>
+		<sheet>
+			<description>You can infuse a shield into a repulsion shield.</description>
+			<!-- <description>Item: A shield (requires attunement)
+			A creature gains a +1 bonus to Armor Class while wielding this shield.
+			The shield has 4 charges. While holding it, the wielder can use a reaction immediately after being hit by a melee attack to expend 1 of the shield’s charges and push the attacker up to 15 feet away. The shield regains 1d4 expended charges daily at dawn.</description> -->
+		</sheet>
+	</element>
+	<element name="Resistant Armor" type="Class Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_CLASS_FEATURE_ARTIFICER_INFUSION_RESISTANT_ARMOR">
+		<supports>Artificer Infusion, TCOE Base</supports>
+		<requirements>[level:6]</requirements>
+		<prerequisite>6th-level artificer</prerequisite>
+		<description>
+			<p><em>Prerequisite: 6th-level artificer</em></p>
+			<p><em>Item: A suit of armor (requires attunement)</em></p>
+			<p>While wearing this armor, a creature has resistance to one of the following damage types, which you choose when you infuse the item: acid, cold, fire, force, lightning, necrotic, poison, psychic, radiant, or thunder.</p>
+		</description>
+		<sheet>
+			<description>You can infuse a suit of armor into a resistant armor.</description>
+			<!-- <description>Item: A suit of armor (requires attunement)
+			While wearing this armor, a creature has resistance to one of the following damage types, which you choose when you infuse the item: acid, cold, fire, force, lightning, necrotic, poison, psychic, radiant, or thunder.</description> -->
+		</sheet>
+	</element>
+	<element name="Returning Weapon" type="Class Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_CLASS_FEATURE_ARTIFICER_INFUSION_RETURNING_WEAPON">
+		<supports>Artificer Infusion, TCOE Base</supports>
+		<description>
+			<p><em>Item: A simple or martial weapon with the thrown property</em></p>
+			<p>This magic weapon grants a +1 bonus to attack and damage rolls made with it, and it returns to the wielder’s hand immediately after it is used to make a ranged attack.</p>
+		</description>
+		<sheet>
+			<description>You can infuse a simple or martial weapon with the thrown property into a returning weapon.</description>
+			<!-- <description>Item: A simple or martial weapon with the thrown property
+			This magic weapon grants a +1 bonus to attack and damage rolls made with it, and it returns to the wielder’s hand immediately after it is used to make a ranged attack.</description> -->
+		</sheet>
+	</element>
 
+	<element name="Replicate Magic Item Table" type="Information" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_CLASS_FEATURE_ARTIFICER_INFUSION_REPLICATE_MAGIC_ITEM_TABLES">
```

Comment:
we should replace ID here to be similar to what we did in ERLW version, along with changing divs that use this ID.